scripts/ci/run-jenkins.sh

```diff
@@ -38,6 +38,8 @@ if [ -x "/usr/bin/dpkg-architecture" ];
 	then
 	EXTRA_CONF_FLAGS="$EXTRA_CONF_FLAGS --host=`/usr/bin/dpkg-architecture -qDEB_HOST_GNU_TYPE`"
 	#force build arch = dpkg arch, sometimes misdetected
+	mkdir -p ~/.mono/config/
+	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.mono/config/
```

Comment:
nit: space missing before |

Is this enough to be able to run the System-btls tests on Linux? If yes, please modify the osx-only condition in run-test-default.sh and enable them on Linux.